build_conf.py

```diff
@@ -250,29 +251,29 @@ def set_work_config(self):
         self.__xt_manifest_uri = 'https://github.com/xen-troops/meta-xt-products.git'
 
         if self.__args.config_file:
-            config = ConfigParser.ConfigParser()
+            config = configparser.ConfigParser()
             config.read(self.__args.config_file)
-            uri = config.get(CFG_SECTION_PATH, CFG_OPTION_WORKSPACE_DIR, 1)
+            uri = config.get(CFG_SECTION_PATH, CFG_OPTION_WORKSPACE_DIR)
```

Comment:
I would propose to maintain behavior as in original (2.7) version. To keep `raw = True` option.
So for python3 it's better to be like
```
uri = config.get(CFG_SECTION_PATH, CFG_OPTION_WORKSPACE_DIR, raw = True)
```
And same below.

Reply:
accept

Reply:
Comment is resolved.